.github/workflows/deploy-and-test-cluster.yml

```diff
@@ -0,0 +1,28 @@
+name: Deploy and Test Cluster
+
+on:
+  push:
+    branches: [main]
```

Comment:
Maybe this could be filtered by path? `paths: - k8s/**` for example.

Reply:
Yeah, that's right, thanks